test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Converters/AcceleratorKeyConverterTests.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using Moq;
+using Xunit;
+
+namespace NuGet.PackageManagement.UI.Test.Converters
+{
+    public class AcceleratorKeyConverterTests
+    {
+        [Theory]
```

Comment:
Another edge case I'd add is when the underscore is at the end of the string. This will ensure any changes to the algorithm preserve the behavior. Example: "Yes_".